tools/wptrunner/wptrunner/wptlogging.py

```diff
@@ -90,6 +90,21 @@ def emit(self, record):
         self.queue.put(data)
 
 
+
+class NullLock:
```

Comment:
Given this is brand new code, can we please actually add types for all of this? This should, I think, match: https://github.com/python/typeshed/blob/614e9499f145e9dccade16d906247276f1d5e6aa/stdlib/_thread.pyi#L16-L20

Can we also add a comment saying that this implements the interface of `threading.RLock`, and probably rename this to `NullRLock`, to clarify this doesn't match `threading.Lock`?